bitnami/harbor/templates/chartmuseum/chartmuseum-cm.yaml

```diff
@@ -0,0 +1,110 @@
+{{- if .Values.chartmuseum.enabled }}
```

Comment:
I can see other components are using the name `<component>-cm-envvars.yaml`, shouldn't you follow this naming conventions for this ConfigMap too?